pages/lyrics/index.vue

```diff
@@ -26,14 +27,15 @@ async function createLyrics() {
   loading.value = true;
 
   try {
-    await new LyricsApi().lyricsPost({
+    const newId = await new LyricsApi().lyricsPost({
```

Comment:
We dont really seem to get an id back here

Reply:
The API changes weren't deployed to Prod yet.